content/updates/serenity-updates-for-the-pim.json

```
@@ -0,0 +1,158 @@
+{
+    "data": [
+        {
+            "startDate": "June, 4th 2020",
+            "notification_duration": 7, 
+            "tags": ["new","updates"],
+            "title": "Easily monitor errors on your connections",
+            "description": "A new page `Monitoring`/`Connections` now lists the last errors of your connections.",
```

```suggestion
            "description": "For each of your connections, a new `Monitoring` page now lists the last integration errors that may have occurred.",
```
@LaureBro The page is not in Monitoring, but in the Connections in the system menu.